### DISCUSSION TITLE
2023-09-20 Weekly Meeting 125 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#418] [General]
Weekly Meeting 124 is happening on Wednesday, September 13, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the :speaker: omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

* new MetaTraversal event coming up [September 28th](https://www.linkedin.com/posts/evonne_vrland-frame-getsocial-activity-7109965645567426560-XQja?utm_source=share&utm_medium=member_desktop)

Continued:

* working group check-ins

Last week: https://github.com/omigroup/omigroup/discussions/417

**Comment by indiebio** [comment]
TODO: work this up to grow the community of people passionate about glTF Physics standards, and people send to their university networks. https://hackmd.io/@indiebio/get-involved-glTF-Physics - MrMetaverse, indiebio, Pearl Hyacinth, Knev.phd

**Comment by mrmetaverse** [comment]
# OMIFest
- the vote is in for our name, its OMIFest 
- dates:
  - thinking october 2024 (that is a year out) 
  - there is a free and open source software fest we could collab with sept 2024 
  - can we do something sooner like this year or the spring of 2024? (No, we are going to do multiple events leading up) 
- we will host multiple little independent events leading up to OMIFest. 
  - OMI Demo Day 9/30/23 
    - Saturday afternoon Eastern time 
    - Demo something connected to OMI Physics 
    - continued in the [next section](https://github.com/omigroup/omigroup/discussions/418#discussioncomment-7060782)
  - multiple ideas in [this thread on discord](https://discord.com/channels/770382203782692945/1149030210503716944/1149030214807064616)

**Reply by indiebio** [reply]
TODO: look at the brainstorm thread on discord to set the dates and topics for the next one or two.

**Comment by mrmetaverse** [comment]
# September 30th OMI Demo Day planning: 
- Saturday afternoon Eastern time 
- Demo something connected to OMI Physics "Interoperable Physics in the Open Metaverse"
  - last week we observed a globe that was using the OMI Physics spec
- have someone share screen and do a lightning talk on a topic 
  - who: whether you implemented OMI Physics, or have a novel / alternative solution, you are welcome to demo physics in the metaverse
  - 5 minutes per speaker (so no one single person hogs the space). 5 min to present your hook. Then we can revisit after with a chat afterwards. 
  - where: we should have this in a virtual world. something was proposed about setting a campfire, with a front page for OMI group. 
- 5 min intro from @mrmetaverse about the open metaverse 
- identify 2-3 people to partake
